### DISCUSSION TITLE
How to use comments in MDX?

### DISCUSSION THREAD

**Discussion by PaulRBerg** [facebook/docusaurus#8823] [Q&A] [answered]
As per this [suggestion](https://github.com/mdx-js/mdx/issues/184#issuecomment-404989493) made by one of the MDX maintainers, adding comments works like this in MDX:

```mdx
# Title

{/* this is a comment */}

Lorem ipsum
```

But Docusaurus will display the above like this:

<img width="186" alt="Screenshot 2023-03-24 at 5 41 34 PM" src="https://user-images.githubusercontent.com/8782666/227573005-c97f0a95-d18a-4dac-8bfd-24ea17a8ecd7.png">

How can we write MDX comments in a Docusaurus website?

**Comment by slorber** [comment, marked as answer]
Docusaurus is still using MDX v1, in the process of upgrading to v2 asap (https://github.com/facebook/docusaurus/pull/8288)


Comments on v1 do not work exactly like in v2

v1 playground: https://mdx-git-renovate-babel-monorepo-mdx.vercel.app/playground/
v2 playground: https://mdxjs.com/playground/

v1 comment outside of JSX? try `<!-- this is a comment -->`
